tests/rt.conf

```diff
@@ -208,8 +208,7 @@ RUN     | datm_cdeps_debug_cfsr
 # ATM-WAV tests                                                                                                                                                                   #
 ###################################################################################################################################################################################
 
-# WW3 not working on Cheyenne in the past, need to check if it works now
-COMPILE | APP=ATMW SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp                                                                        | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
-RUN     | fv3_gfdlmprad                                                                                                           | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
-RUN     | fv3_gfdlmprad_atmwav                                                                                                    | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
+COMPILE | APP=ATMW SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp                                                                        |                                         | fv3 |
+RUN     | fv3_gfdlmprad                                                                                                           |                                         | fv3 |
+RUN     | fv3_gfdlmprad_atmwav                                                                                                    |                                         | fv3 |
 RUN     | fv3_wrtGauss_nemsio_c768                                                                                                | +               hera.intel orion.intel  | fv3 |
```

Comment:
fv3_wrtGauss_nemsio_c768

Reply:
Okay, do I remove it in this commit then?

Reply:
yes, we should remove the 768 orion test.

Reply:
Just on orion, or completely?